src/components/input/index.tsx

```diff
@@ -120,7 +120,7 @@ const Input = forwardRef<HTMLInputElement, Props>(
       return () => {
         debouncedChangeHandler?.cancel()
       }
-    }, [])
+    }, [debouncedChangeHandler])
```

Comment:
Alternatively, we can also use `useDeepCompareEffect`

Reply:
as discussed, we will disable the rule for this component